src/patterns/validation/index.md.njk

```diff
@@ -39,14 +39,14 @@ Validation should refuse to accept:
 
 For example, if you’re asking for someone’s date of birth you should not accept ‘13’ in the month field.
 
-Validation should help remove (or ignore) any spaces or invisible characters, unless it makes the information too ambiguous for you to use.
+Use validation to ignore, or remove, unwanted characters, unless it would make the information too unclear for you to use. For example, any spaces, invisible characters or punctuation, like hyphens, brackets, dashes and full stops.
```

Comment:
```suggestion
Use validation to ignore unwanted characters, unless it would make the information too unclear for you to use. For example, any spaces, invisible characters or punctuation, like hyphens, brackets, dashes and full stops.
```